src/test/java/com/groupseven/pdfproject/AppTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.groupseven.pdfproject;
+
+import javafx.stage.Stage;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author dwkal
+ */
+public class AppTest {
+
+    public AppTest() {
+    }
+
+    @BeforeClass
+    public static void setUpClass() {
+    }
+
+    @AfterClass
+    public static void tearDownClass() {
+    }
+
+    @Before
+    public void setUp() {
+    }
+
+    @After
+    public void tearDown() {
+    }
+
+    /**
+     * Test of start method, of class App.
+     */
+    @Test
+    public void testStart() throws Exception {
+        System.out.println("start");
+        Stage primaryStage = null;
+        App instance = new App();
+        instance.start(primaryStage);
```

Comment:
This class may not make the most sense to test this way, and it may be one of the classes we omit from junit.

Starting a GUI doesn't really make sense with JUnit tests, and it opens a window on my local machine and then errors when I close it causing it to fail